Makefile

```diff
@@ -4,6 +4,8 @@ INST_LUADIR ?= $(INST_PREFIX)/share/lua/5.1
 INST_BINDIR ?= /usr/bin
 INSTALL ?= install
 UNAME ?= $(shell uname)
+OR_EXEC ?= $(shell which openresty)
+LUA_JIT_DIR ?= $(shell TMP='./v_tmp' && $(OR_EXEC) -V &>$${TMP} && cat $${TMP} | grep prefix | grep -Eo 'prefix=(.*?)/nginx' | grep -Eo '/.*/' && rm $${TMP})luajit
```

Comment:
'./v_tmp' -> '/tmp/v_tmp', how about this?

Reply:
it doesn't make any sense, the `v_tmp` file will been clean immediately at the end of this command.